src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -26,7 +27,7 @@
  *
  * @final since version 3.4
```

Comment:
`@final` is telling the intention to external users - extending internally is not an issue to me.